server/protocols/jmap-integration-testing/cassandra-jmap-integration-testing/src/test/java/org/apache/james/jmap/cassandra/CassandraSetMessagesMethodTest.java

```diff
@@ -41,4 +47,9 @@ protected void await() {
         rule.await();
     }
     
+    @Override
+    protected MessageId randomMessageId() {
+        return TestMessageId.of(random.nextInt(1000000) + 100);
```

Comment:
There's no constraint for random messageId: they should not be ordered for example...
We are just ensuring that we take an arbitrary id for 100 to 1000100.